plugins/inputs/all/all.go

```diff
@@ -208,4 +208,5 @@ import (
 // 	_ "github.com/influxdata/telegraf/plugins/inputs/zfs"
 // 	_ "github.com/influxdata/telegraf/plugins/inputs/zipkin"
 // 	_ "github.com/influxdata/telegraf/plugins/inputs/zookeeper"
+	_ "github.com/influxdata/telegraf/plugins/inputs/ah_wireless"
```

Comment:
How much does this plugin increase the size of the executable?  We have size constraints on our EXOS summitlite_arm platform.

Reply:
@arnelplim 
Without plugin 13039575
With plugin 13222735

So its almost 200Kb of increment.
Is that ok? How much is the size constraint?

Reply:
200kb might be a problem.  EXOS and VOSS doesn't need this plugin, so can we have it commented out by default and adjust your build adjust to include it for wireless platform only?

Reply:
@vullanatt Thoughts?  I think 200k will be an issue on summitlite_arm, right?

Reply:
> 200kb might be a problem. EXOS and VOSS doesn't need this plugin, so can we have it commented out by default and adjust your build adjust to include it for wireless platform only?

This makes sense since, this plugin wont work in any other platform apart from hive